BugTrackerExample/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "Stateless": {
+      "target": "project",
+      "version": "2.5.0-*"
```

Comment:
`"version"` doesn't need to be specified in a `"target": "project"` dependency (I don't think it hurts, but it'll get out of sync as the main assembly's version evolves).